hindi/priyanka-chopra

```diff
@@ -0,0 +1 @@
+Priyanka Chopra is an Indian film actress and singer, and the winner of the Miss World pageant of 2000. (https://en.wikipedia.org/wiki/Priyanka_Chopra)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.